### DISCUSSION TITLE
How to make changes to the UI? If I am running a para?

### DISCUSSION THREAD

**Discussion by pavankm99** [Erudika/scoold#336] [Q&A]
Hello wanted to check, how to make changes to any UI component if I want to add a different logo etc. I see the code is under `assets` and it is serving from `jquery.min.js` wanted to check if there is a repo or a steps to make changes and reflect on the local.

**Comment by albogdano** [comment]
There are lots of configuration options for that purpose, described here - https://github.com/Erudika/scoold#customizing-the-ui.
